youtube_dl/extractor/abc.py

```diff
@@ -103,6 +103,55 @@ def _real_extract(self, url):
         }
 
 
+class ABCIViewShowIE(InfoExtractor):
+    IE_NAME = 'abc.net.au:iview:show'
+    _VALID_URL = r'https?://iview\.abc\.net\.au/(?:[^/]+/)*show/(?P<id>[^/?#]+)/?$'
```

Comment:
No trailing $.

Reply:
This is required to avoid colliding with the single-episode URL format.

Reply:
As already said: no trailing $. Override `suitable`.

Reply:
@DanielHeath try adding this to ABCIViewShowIE instead of using the trailing $.

`
    @classmethod
    def suitable(cls, url):
        return False if ABCIViewIE.suitable(url) else super(ABCIViewShowIE, cls).suitable(url)
`

Reply:
I've added this as a suggestion: https://github.com/ytdl-org/youtube-dl/pull/16755/files#r278522770

This should work in place of https://github.com/ytdl-org/youtube-dl/pull/16755#discussion_r262000428

Reply:
@dstftw I've removed the trailing $ in place of a `suitable` method.

Reply:
@dstftw I've tested this and it works as expected using the `suitable` method